components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -72,6 +74,7 @@
 #include "content/public/browser/service_process_host.h"
 #include "content/public/browser/storage_partition.h"
 #include "content/public/browser/url_data_source.h"
+#include "extensions/common/constants.h"
```

Comment:
why was this added?

Reply:
Thanks, that was an oversight  - removed.